common/src/main/scala/com/gu/media/youtube/YouTubeAccess.scala

```diff
@@ -49,10 +51,23 @@ trait YouTubeAccess extends Settings {
     .build
     .setRefreshToken(refreshToken)
 
+  private val youTubeRequestLogger = new YouTubeRequestInitializer{
+    override def initializeYouTubeRequest(request: YouTubeRequest[_]): Unit = {
+      super.initializeYouTubeRequest(request)
+      val markers: LogstashMarker = Markers.appendEntries(Map(
```

Comment:
Should we log `POST` body parameters as well? Only if they are used by the YouTube SDK and are readable of course

Reply:
I've tried to log everything in the [YouTubeRequest](https://developers.google.com/resources/api-libraries/documentation/youtube/v3/java/latest/com/google/api/services/youtube/YouTubeRequest.html#getPrettyPrint--) that looks useful, we're logging the `httpContent`, if there something I've missed I'm happy to add it.